### DISCUSSION TITLE
Copy after backup?

### DISCUSSION THREAD

**Discussion by roliva6** [olahallengren/sql-server-maintenance-solution#634] [Questions & Answers]
Hello,
Mirroring isn't available on Standard Edition, so what methods do people use if they want a "backup of the backup?" Is there a way to easily integrate with the maintenance solution?
Thanks in advance!

**Comment by jacobgexigo** [comment]
The simplest method would be to use RoboCopy to copy the file to the second location.  If the location is Azure BLOB storage, you can use AzCopy in PowerShell.

**Comment by AdrianTroy** [comment]
Although Azcopy and Powershell are good to backup to clouds , but I prefer using GUI alternatives like Gs Richcopy 360 and Syncback pro

**Comment by bmciarimboli** [comment]
Is it possible to call AZCOPY or ROBOCOPY automatically when the backup completes from the maintenance solution? @olahallengren
